packit/api.py

```diff
@@ -192,6 +192,11 @@ def sync_release(
                 f"Upstream commit: {self.up.local_project.git_repo.head.commit}\n"
             )
 
+            path = os.path.join(self.dg.local_project.working_dir, "README.packit")
+            logger.debug(f"Path of README {path}")
+            with open(path, "w") as f:
```

Comment:
when the file exists already, we'll overwrite it

is that something we want?

I really intend to discuss.

Reply:
I think so.

- :heavy_plus_sign: we can easily change the content in the future
- :heavy_plus_sign: we get the time of the last packit operation in the file metadata